doc/about/update_notifications/index.md

```diff
@@ -0,0 +1,58 @@
+# Update Notifications {#about_update_notifications}
+
+[TOC]
+
+Outdated software is a huge issue in science workflows. Especially when the software is not part of the packaging
+provided by the operating system, it tends to be installed once and never updated.
+
+We provide update notifications as a convenient service to app users so that they immediately see when a new version is
+available. When we assume that an application is being run by its developer we nag the developer to also keep their
+checkout of the Sharg library up-to-date. This ensures new features get to app developers faster, applications have
+fewer bugs, and we receive fewer bug reports on outdated versions.
+
+## How it works
+
+Upon starting an application using the `sharg::argument_parser`, it is determined whether the app is built in release
+mode (we assume a user is running it) or in debug mode (we assume the developer is running it).
+
+We then send a web-request to our server with the following:
+
+ * The application name and its version
+ * The Sharg version that the application was built with
+ * The operating system type (Linux, macOS, Windows or BSD)
+ * The CPU type (32 or 64bit)
+
+But we send at most one request per day [we keep track of this by saving a timestamp to `.config/seqan/app_name`].
+
+If a newer app version is registered in our database (or can be automatically determined), we inform the user about it.
+If the app is running in debug mode and the Sharg version is outdated, we assume the user is the app developer and
+inform them about it. Additionally, we recommend to register your app with us 
+if it is not already registered (support@seqan.de).
+
+## Privacy Implications
+
+Only the information listed above is transmitted; no data is gathered about the usage of the app, and, of course, no
+information regarding your input/output files is transmitted. All our code is open source so this can be verified at
+any time.
```

Comment:
```suggestion
information regarding your input/output files is transmitted. All our code is open source, and hence this can be verified at any time.
```